flink-ml-core/src/test/java/org/apache/flink/ml/api/StageTest.java

```diff
@@ -388,5 +388,13 @@ public void testValidators() {
         ParamValidator<Integer> notNull = ParamValidators.notNull();
         Assert.assertTrue(notNull.validate(5));
         Assert.assertFalse(notNull.validate(null));
+
+        ParamValidator<Object[]> nonEmptyArray = ParamValidators.nonEmptyArray();
+        Assert.assertTrue(nonEmptyArray.validate(new String[] {"1"}));
+        Assert.assertFalse(nonEmptyArray.validate(null));
+        Assert.assertFalse(nonEmptyArray.validate(new String[0]));
+
+        ParamValidator<Integer> nonEmptyArrayInteger = ParamValidators.nonEmptyArray();
```

Comment:
hmm.. should its type be `ParamValidator<Integer[]>`?

Reply:
I wanted to test that the ParamValidator should fail of its generic is not an array type. I found that it can be achieved by 
```java
public static <T> ParamValidator<T[]> nonEmptyArray()
```
and get the error at compile time. In this way I don't need the `ParamValidator<Integer>` check anymore. I'll remove it.